accession-commons-monotonic-generator-jpa/src/test/java/uk/ac/ebi/ampt2d/commons/accession/generators/DecoratedAccessionGeneratorTest.java

```diff
@@ -134,8 +135,8 @@ public void testGenerateSuffix() throws Exception {
     private MonotonicAccessionGenerator<String> getGenerator() throws Exception {
         assertEquals(0, repository.count());
 
-        MonotonicAccessionGenerator<String> generator = new MonotonicAccessionGenerator<>(BLOCK_SIZE, CATEGORY_ID,
-                INSTANCE_ID, service);
+        MonotonicAccessionGenerator<String> generator = new MonotonicAccessionGenerator<>(BLOCK_SIZE,
+                NEXT_BLOCK_INTERVAL, CATEGORY_ID, INSTANCE_ID, service);
```

Comment:
NEXT_BLOCK_INTERVAL should not be globally configured. This should be configurable for each category following the same approach as for the Init value.